### PR TITLE
changes to currently running dewiki-test13

### PR DESCRIPTION
**Pull request by KaiNissen**

### CODE REVIEW

File: desktop/banner_var.js [hunk 1]
```diff
@@ -22,15 +22,20 @@ const DevGlobalBannerSettings = require( '../shared/global_banner_settings' );
 const GlobalBannerSettings = window.GlobalBannerSettings || DevGlobalBannerSettings;
 const Translations = {}; // will only be needed for English banner, German defaults are in DesktopBanner
 const BannerFunctions = require( '../shared/banner_functions' )( GlobalBannerSettings, Translations );
-const campaignDays = new CampaignDays(
-	startOfDay( GlobalBannerSettings[ 'campaign-start-date' ] ),
-	endOfDay( GlobalBannerSettings[ 'campaign-end-date' ] )
+const campaignDaySentence = new CampaignDaySentence(
+	new CampaignDays(
```
Comment (gbirke) [inline desktop/banner_var.js:26]: Any reason why you don't want to have a shared instance of `campaignDays`?

**Review by gbirke — COMMENTED** [review]
I don't understand the title of this PR - Is this a new test or do existing banners that are deployed change?

File: desktop/banner_var.js (hunk 1 above)
Comment (KaiNissen) [reply desktop/banner_var.js:26]: Yes, the "start date" is different for the projection (`donations-date-base`).

**Comment by KaiNissen** [comment]
This PR contains changes that were introduced on the 2nd day (e. g. 
[add sentence comparing donors and impressions](https://github.com/wmde/fundraising-banners/pull/50/commits/8aec8a64781e9f1a2f9b4d7c0c56684cd699ec83)) or even later.

**Comment by wiese** [comment]
+0,5 This is becoming kind of complex.

File: desktop/banner_var.js (hunk 1 above)
Comment (gbirke) [reply desktop/banner_var.js:26]: Ah yes, did not see that. Thanks!

**Pull request merged**